api-specificatie/openapi.yaml

```diff
@@ -1440,10 +1440,10 @@ components:
         adres:
           $ref: '#/components/schemas/AdresHalBasis'
           description: 'De aan de nummeraanduiding gerelateerde adres (object).'
-        ligtAanOpenbareRuimte:
+        openbareRuimte:
           $ref: '#/components/schemas/OpenbareRuimteHalBasis'
           description: 'De aan de nummeraanduiding gerelateerde openbareruimte (object(en)).'
-        ligtInWoonplaats:
+        woonplaats:
           $ref: '#/components/schemas/WoonplaatsHalBasis'
           description: 'De aan de nummeraanduiding gerelateerde woonplaats (object(en)).'
```

Comment:
Een nummeraanduiding kan maar in één woonplaats liggen, meervoudsvorm mag dus weg.

Reply:
Ook hier neem ik aan dat jouw opmerking over de description gaat.

Reply:
Ja.